charts/openmetadata/templates/service.yaml

```diff
@@ -3,11 +3,27 @@ kind: Service
 metadata:
   name: {{ include "OpenMetadata.fullname" . }}
   labels:
-    {{- include "OpenMetadata.labels" . | nindent 4 }}
-  {{- with .Values.service.annotations }}
+    app.kubernetes.io/name: '{{ include "OpenMetadata.name" .}}'
```

Comment:
I would recommend using [helper class](https://github.com/open-metadata/openmetadata-helm-charts/blob/fd71bcc9eab12eebc13296f7b3f4a3a1f98dd392/charts/openmetadata/templates/_helpers.tpl) to define labels starting with `app.kubernetes.io/*`!

Reply:
used helper class to define labels starting with app.kubernetes.io/*